pages/docs/tutorials/native/apple-framework.md

```diff
@@ -138,6 +142,44 @@ Most of the collection types are mapped to similar collection types from the oth
 {:.wide.zebra}
 
 
+### Kotlin Numbers and NSNumber
+
+The next part of the `<Framework>/Headers/Demo.h` contains number type mappings
+between Kotlin/Native and `NSNumber`. We have the base class called `DemoNumber` in Objective-C
+and `KotlinNumber` in Swift. It extends `NSNumber`.
+There are also child classes per Kotlin number type:
+
+|Kotlin|Swift|Objective-C| Simple type |
+-------|-----|-----------|
+|`-`      |`KotlinNumber` |`<Package>Number` | `-` |
+|`Byte`   |`KotlinByte`   |`<Package>Byte`   | `char` |
+|`UByte`  |`KotlinUByte`  |`<Package>UByte`  | `unsigned char` |
+|`Short`  |`KotlinShort`  |`<Package>Short`  | `short` |
+|`UShort` |`KotlinUShort` |`<Package>UShort` | `unsigned short` |
+|`Int`    |`KotlinInt`    |`<Package>Int`    | `int` |
+|`UInt`   |`KotlinUInt`   |`<Package>UInt`   | `unsigned int` |
+|`Long`   |`KotlinLong`   |`<Package>Long`   | `long` |
+|`ULong`  |`KotlinULong`  |`<Package>ULong`  | `unsigned long` |
```

Comment:
Kotlin `(U)Long` corresponds to `(unsigned) long long` C type.

Reply:
Fixed. Where do we map 
```
- (instancetype)initWithLong:(long)value __attribute__((unavailable));
- (instancetype)initWithUnsignedLong:(unsigned long)value __attribute__((unavailable));
```

Reply:
We do not map these ones as Kotlin doesn't have a concept like C `long` type.
The same is true for `initWith(Unsigned)Integer` with `NS(U)Integer` types.